prov/verbs/src/fi_verbs.h

```diff
@@ -107,12 +111,6 @@
 
 extern struct fi_provider fi_ibv_prov;
 
-extern size_t verbs_default_tx_size;
-extern size_t verbs_default_rx_size;
-extern size_t verbs_default_tx_iov_limit;
-extern size_t verbs_default_rx_iov_limit;
-extern size_t verbs_default_inline_size;
-
 extern size_t verbs_min_rnr_timer;
```

Comment:
I guess this would have to be removed as well.

Reply:
Sure, thank!